### PR TITLE
[front] feat: add uTip on the donate page

### PR DESCRIPTION
**Pull request by GresilleSiffle**

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -183,9 +183,12 @@
...
     "openSourceContributors": "As Tournesol is an open source project, we have been lucky to benefit from contributions by multiple volunteers. Find our wonderful contributors on <2>GitHub</2>",
-    "donateWeAreASmallTeam": "Because we are a small team of mostly volunteers, the development of Tournesol is slower than we would like it to be. If you can, please consider helping us, through coding or through donations. Check-out our <2>open source code</2>, or join our <4>Discord</4>.",
+    "donateWeAreASmallTeam": "Because we are a small team of mostly volunteers, the development of Tournesol is slower than we would like it to be. If you can, please consider helping us with a donation using one of methods below. You can also help us by contributing to our open source code. Check-out our <2>open source code</2>, or join our <5>Discord</5>.",
     "donateHowTo": "How to make a donation?",
-    "donateByDirectTransfer": "By direct transfer",
+    "donateWithUtipTitle": "With Utip",
+    "donateWithUtipDescription": "Utip is an online crowdfunding platform. Visit our <2>Utip page</2> to make a one-time or recurring donation.",
```
Comment (amatissart) [inline frontend/public/locales/en/translation.json:189]: It seems that the official spelling is "uTip"
```suggestion
    "donateWithUtipTitle": "With uTip",
    "donateWithUtipDescription": "uTip is an online crowdfunding platform. Visit our <2>uTip page</2> to make a one-time or recurring donation.",
```

File: frontend/public/locales/fr/translation.json [hunk 2]
```diff
@@ -187,9 +187,12 @@
...
-    "donateWeAreASmallTeam": "Du fait que notre équipe soit petite, composée majoritairement de bénévole, le développement de Tournesol est plus lent que ce que nous voudrions. Si vous en avez la possibilité, n'hésitez pas à nous aider avec un don, ou des contributions au code source. Consultez notre <2>code source libre ici</2>, et rejoignez nous sur <4>Discord</4>.",
+    "donateWeAreASmallTeam": "Du fait que notre équipe soit petite, composée majoritairement de bénévole, le développement de Tournesol est plus lent que ce que nous voudrions. Si vous en avez la possibilité, n'hésitez pas à nous aider avec un don en utilisant une des méthodes ci-dessous. Vous pouvez aussi nous aider en participant au developpement de notre code open source. Consultez notre <2>code source libre ici</2>, et rejoignez nous sur <5>Discord</5>.",
     "donateHowTo": "Comment faire un don ?",
-    "donateByDirectTransfer": "Par virement direct",
+    "donateWithUtipTitle": "Avec Utip",
+    "donateWithUtipDescription": "Utip est une plateforme de financement participatif. Visitez notre <2>page Utip</2> pour faire un don unique ou recurrent.",
```
Comment (amatissart) [inline frontend/public/locales/fr/translation.json:193]: Same in french
```suggestion
    "donateWithUtipTitle": "Avec uTip",
    "donateWithUtipDescription": "uTip est une plateforme de financement participatif. Visitez notre <2>page uTip</2> pour faire un don unique ou recurrent.",
```

**Review by amatissart — COMMENTED** [review]
Looks good to me.

Perhaps the link to the uTip page could be made a bit more obvious. E.g. by making the title and the image clickable too, or by adding a button "Donate on uTip"?

**Comment by lfaucon** [comment]
I made the whole card clickable with the mouse as a cursor on it. What do you think @amatissart ?

**Review by amatissart — APPROVED** [review]

**Pull request merged**